README.md

```diff
@@ -51,16 +51,24 @@
 > Some key feature of this project
 > Features
 
-- **User can ....**
-- **User can ....**
+<div align="flex">
+
+  ![image1](https://user-images.githubusercontent.com/88631022/217943844-58ccc9f7-6675-45fd-9d7e-72bac76c355c.PNG)
+  
+  ![image2](https://user-images.githubusercontent.com/88631022/217943918-dc4cfcb3-eeb1-4a32-83ab-9465a940dd13.PNG)
+</div>
+
+- **User can switch countries and view details about countries by just clicking on it**
+- **User can search among countries**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🚀 Live Demo <a name="live-demo"></a>
+> ## 🚀 Live Demo <a name="live-demo"> </a>
+- [Check out the live demo here](https://explore-america.onrender.com)
 
 > ## Project demo
->
-> Site is under construction and the Project Demo will update soon.
+[Check out the live video here](https://www.loom.com/share/678ae641b5fe4d64b231a424bcfefbbc)
+
```

Comment:
Well done for making the video presentation, however, as part of the project requirements, you should also highlight some interesting piece of code or something you built that you are very proud of  in the video. Kindly check and update that. :wink:

Reply:
- [ ] This issue is well resolved :heavy_check_mark: